tests/integration/relations/test_db_admin.py

```diff
@@ -163,3 +167,5 @@ async def test_create_db_admin_legacy_relation(ops_test: OpsTest):
             pg_user=pgb_user,
             pg_user_password=pgb_password,
         )
+
+        logger.info(await get_pgb_log(ops_test))
```

Comment:
```suggestion
```

Reply:
These logging lines are actually there intentionally, so we can see what pgbouncer is doing when something goes wrong - it makes the test logs a bit longer, but I think it makes them more useful.